app/presenters/claim_csv_presenter.rb

```diff
@@ -96,7 +96,11 @@ def current_or_end_state
   end
 
   def state_reason_code
-    @journey.last.reason_code
+    @journey.last.reason_code.flatten.join(', ')
```

Comment:
you could put a .humanize in here for the report, if this is the output for the report?! to change no_indictment to "No indictment" etc. or perhaps a `ClaimStateTransitionReason.get()` call for each

```
@journey.last.reason_code.humanize.flatten.join(', ')
```


```
@journey.last.reason_code.flat_map do |reason| 
  ClaimStateTransitionReason.get(reason) 
end.join(',')
```

Reply:
They've always had the raw values in the MI report, I'm just showing multiples now!